jobs/nfsbroker-bbr-lock/templates/metadata.sh.erb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "---
+backup_should_be_locked_before:
+- job_name: uaa
+  release: uaa
+- job_name: credhub
```

Comment:
What happens if the `credhub` job is missing in the deployment?

Reply:
Nothing, it will be as if nfsbroker is not dependent on credhub